core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -153,19 +204,247 @@ describe('Learner dashboard functionality', function() {
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.navigateToSkillsTab();
+    await topicsAndSkillsDashboardPage.assignSkillToTopic(
+      'Learner Dashboard Skill 1', TOPIC_NAME);
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.navigateToTopicWithIndex(0);
+    await topicEditorPage.addSubtopic(
+      'Learner Dashboard Subtopic 1', 'ld-subtopic-one',
+      '../data/test2_svg.svg', 'Subtopic content');
+    await topicEditorPage.saveTopic('Added subtopic.');
+
+    await topicEditorPage.navigateToTopicEditorTab();
+    await topicEditorPage.navigateToReassignModal();
+
+    await topicEditorPage.dragSkillToSubtopic('Learner Dashboard Skill 1', 0);
+    await topicEditorPage.saveRearrangedSkills();
+    await topicEditorPage.saveTopic('Added skill to subtopic.');
+
+    await topicEditorPage.updateMetaTagContent('meta tag content');
+    await topicEditorPage.updatePageTitleFragment('fragment');
+    await topicEditorPage.saveTopic('Added meta tag and page title fragment.');
+
+    await topicEditorPage.publishTopic();
+    /**  There is one topic on the server named Learner Dashboard Topic 1
+     * which is linked to a subtopic named Learner Dashboard Subtopic 1
+     * and a story called Story Title. Learner Dashboard Subtopic 1 has one
+     * skill in it named Learner Dashboard Skill 1.
+     */
+    await learnerDashboardPage.get();
+    await learnerDashboardPage.navigateToHomeSection();
+    await learnerDashboardPage.expectNumberOfTopicsInSuggestedForYou(1);
+    await learnerDashboardPage.expectNumberOfTopicsInContinueWhereYouLeftOff(0);
+    await learnerDashboardPage.navigateToGoalsSection();
+    await learnerDashboardPage.expectNameOfTopicInEditGoalsToMatch(
+      TOPIC_NAME);
+    await learnerDashboardPage.addTopicToLearnerGoals();
+    await learnerDashboardPage.navigateToGoalsSection();
+    await learnerDashboardPage.expectNameOfTopicInCurrentGoalsToMatch(
+      `Learn ${TOPIC_NAME}`);
+    await learnerDashboardPage.navigateToHomeSection();
+    await learnerDashboardPage.expectNumberOfTopicsInContinueWhereYouLeftOff(1);
+    await users.logout();
+  });
+
+  it('should display all the topics that are partially learnt or learnt ' +
+    'in skill proficiency section, learnt topics in completed goals section ' +
+    'and completed stories in completed stories section', async function() {
+    var TOPIC_NAME = 'Learner Dashboard Topic 2';
+    var TOPIC_URL_FRAGMENT_NAME = 'ld-topic-two';
+    var TOPIC_DESCRIPTION = 'Topic description';
+    await users.createAndLoginCurriculumAdminUser(
+      'creator@learnerDashboard2.com', 'learnerDashboard2');
+    await createDummyExplorations();
+    var handle = await browser.getWindowHandle();
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.createTopic(
+      TOPIC_NAME, TOPIC_URL_FRAGMENT_NAME, TOPIC_DESCRIPTION, false);
```

Comment:
Ditto. Should above expectation be applied?

Reply:
Done.